### PR TITLE
fix: multiple custom transport cleanup

### PR DESCRIPTION
**Pull request by shanejonas**

### CODE REVIEW

File: src/hooks/useTransport.ts [hunk 1]
```diff
@@ -74,6 +74,7 @@ const getTransportFromType = async (
       }
       public close() {
         intermediateTransport.unsubscribe();
+        intermediateTransport.close();
```
Comment (zcstarr) [inline src/hooks/useTransport.ts:77]: is it better to send then close the transport?

File: src/hooks/useTransport.ts [hunk 2]
```diff
@@ -72,9 +72,9 @@ const getTransportFromType = async (
           },
         });
       }
-      public close() {
+      public async close() {
```
Comment (shanejonas) [inline src/hooks/useTransport.ts:75]:
```suggestion
      public close() {
```

**Review by zcstarr — APPROVED** [review]
🚢

**Pull request merged**